shards/sched.go

```diff
@@ -227,3 +297,25 @@ func (t *deadlineTimer) Stop() {
 	t.t.Stop()
 	t.t = nil
 }
+
+// parseTuneables parses a comma separated string of key=value pairs. "=value"
+// is optional, defaults to 1. value is expected to be an int. Errors are
+// ignored (value will be 0).
+func parseTuneables(v string) map[string]int {
+	m := map[string]int{}
+
+	for _, kv := range strings.Split(v, ",") {
+		if kv == "" {
+			continue
+		}
+
+		p := strings.SplitN(kv, "=", 2)
+		if len(p) == 1 {
+			m[p[0]] = 1
+		} else {
+			m[p[0]], _ = strconv.Atoi(p[1])
+		}
+	}
+
+	return m
+}
```

Comment:
Do need to worry about trimming spaces?

Reply:
I thought about that, but don't want to overengineer the parser. I think it being strict + the logging when non-defaults are used should be good enough. WDYT?